<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="general_tab">
      <attribute name="title">
       <string>&amp;General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="appearance_group">
         <property name="title">
          <string>Appearance</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="skin_layout">
            <item>
             <widget class="QRadioButton" name="font_rbtn">
              <property name="text">
               <string>font</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="select_font_btn">
              <property name="text">
               <string notr="true">...</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="skin_spacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QRadioButton" name="skin_rbtn">
              <property name="text">
               <string>skin</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="skin_cbox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QCheckBox" name="is_separator_flashes">
              <property name="text">
               <string>flashing separator</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="scaling_label">
              <property name="text">
               <string>Scaling:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="scaling_x_edit">
              <property name="toolTip">
               <string>horizontal scaling</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="suffix">
               <string notr="true">%</string>
              </property>
              <property name="minimum">
               <number>10</number>
              </property>
              <property name="maximum">
               <number>800</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="scaling_same_btn">
              <property name="toolTip">
               <string>same scaling</string>
              </property>
              <property name="text">
               <string notr="true">=</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="scaling_y_edit">
              <property name="toolTip">
               <string>vertical scaling</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="suffix">
               <string notr="true">%</string>
              </property>
              <property name="minimum">
               <number>10</number>
              </property>
              <property name="maximum">
               <number>800</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="opacity_layout">
            <item>
             <widget class="QLabel" name="opacity_label">
              <property name="text">
               <string>&amp;opacity</string>
              </property>
              <property name="buddy">
               <cstring>opacity_slider</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="opacity_slider">
              <property name="minimum">
               <number>5</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="opacity_edit">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="suffix">
               <string notr="true">%</string>
              </property>
              <property name="minimum">
               <number>5</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="colorization_layout_1">
            <item>
             <widget class="QCheckBox" name="use_colorization">
              <property name="text">
               <string>render the clock with a tint of selected &amp;color</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="select_colorization_color">
              <property name="toolTip">
               <string>colorization color</string>
              </property>
              <property name="text">
               <string notr="true">...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="colorization_layout_2">
            <item>
             <widget class="QLabel" name="colorization_strength_label">
              <property name="text">
               <string>stre&amp;ngth</string>
              </property>
              <property name="buddy">
               <cstring>colorization_strength_slider</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="colorization_strength_slider">
              <property name="minimum">
               <number>5</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="colorization_strength_edit">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="suffix">
               <string notr="true">%</string>
              </property>
              <property name="minimum">
               <number>5</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="time_zone_layout">
         <item>
          <widget class="QCheckBox" name="use_time_zone">
           <property name="text">
            <string>time &amp;zone</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="time_zone_edit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttons_layout">
     <item>
      <widget class="QPushButton" name="import_btn">
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="export_btn">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>font_rbtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>select_font_btn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>skin_rbtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>skin_cbox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_time_zone</sender>
   <signal>toggled(bool)</signal>
   <receiver>time_zone_edit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>opacity_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>opacity_edit</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>opacity_edit</sender>
   <signal>valueChanged(int)</signal>
   <receiver>opacity_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorization_strength_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>colorization_strength_edit</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorization_strength_edit</sender>
   <signal>valueChanged(int)</signal>
   <receiver>colorization_strength_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_colorization</sender>
   <signal>toggled(bool)</signal>
   <receiver>select_colorization_color</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_colorization</sender>
   <signal>toggled(bool)</signal>
   <receiver>colorization_strength_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_colorization</sender>
   <signal>toggled(bool)</signal>
   <receiver>colorization_strength_slider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_colorization</sender>
   <signal>toggled(bool)</signal>
   <receiver>colorization_strength_edit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
